pysar/ifgram_inversion.py

```diff
@@ -583,15 +611,15 @@ def subsplit_boxes_by_num_workers(box, num_workers, dimension='y'):
 
     if dimension == 'y':
         y_diff = y1 - y0
-        for i in range(num_workers):
-            start = (i * y_diff) // num_workers
-            end = ((i + 1) * y_diff) // num_workers if i + 1 != num_workers else y_diff
+        for i in range(number_of_splits):
+            start = (i * y_diff) // number_of_splits
+            end = ((i + 1) * y_diff) // number_of_splits if i + 1 != number_of_splits else y_diff
```

Comment:
Could you add some comments on what the `start` and `end` computations are? They are not immediately clear to me.

Reply:
Hmm, let's talk about this in person. I had a hard time coming up with a comment so I'm wondering when I explain it to you in person, which part was the missing thing. I can then include that in the comment